src/Init/Data/SInt/Basic.lean

```diff
@@ -367,6 +379,12 @@ def Int32.neg (i : Int32) : Int32 := ⟨⟨-i.toBitVec⟩⟩
 
 instance : ToString Int32 where
   toString i := toString i.toInt
+instance : Repr Int16 where
+  reprPrec i prec := reprPrec i.toInt prec
+instance : ReprAtom Int16 := ⟨⟩
```

Comment:
@TwoFX Hi!
I've accidentally stubbed upon this change, and based on the other additions in this PR, it seems it should be `Int32` instead of `Int16` here

Reply:
Thanks! Fixed in #7467.